packages/flutter/lib/src/material/data_table.dart

```diff
@@ -393,7 +393,13 @@ class DataTable extends StatelessWidget {
     this.onSelectAll,
     this.decoration,
     this.dataRowColor,
+    @Deprecated(
+      'This property is no longer used, please use dataRowMinHeight and dataRowMaxHeight instead. '
+      'This feature was deprecated after v3.5.0-10.0.pre.',
+    )
```

Comment:
This is still being used and is backwards compatible during the deprecation period, can you update all of the notices to reflect that?

```suggestion
    @Deprecated(
      'Migrate to use dataRowMinHeight and dataRowMaxHeight instead. '
      'This feature was deprecated after v3.5.0-10.0.pre.',
    )
```